filter-effects-2/Overview.bs

```diff
@@ -50,28 +54,281 @@ If the value of the 'backdrop-filter' property is ''backdrop-filter/none'' then
 filter effect applied. Otherwise, the list of functions are applied in the
 order provided.
 
-The first filter function or <a element>filter</a> reference in the list takes
-the element's BackgroundImage as the input image. Subsequent
+## Filtering and Clipping ## {#backdrop-filter-operation}
+
+An element (call it B) with a backdrop-filter property other than ''backdrop-filter/none'' is conceptually rendered as if
+the following steps are performed:
+1. Copy the <a>Backdrop Root Image</a> into a temporary buffer, such as a raster image. Call this buffer T’.
+2. Apply the backdrop-filter’s filter operations to the entire contents of T'.
+3. If element B has any transforms, apply the <b>inverse</b> of those transforms to the contents of T’.
+4. Apply a clip to the contents of T’, using the border box of element B, included <a href="https://drafts.csswg.org/css-backgrounds-3/#border-radius">border-radius</a> if specified. Note that the children of B are not considered for the sizing or location of this clip.
+5. Draw all of element B, including its background, border, and any children elements, into T’.
+6. If element B has any transforms, effects, or clips, apply those to T’.
+7. Composite the contents of T’ into element B's parent, using source-over compositing.
+
+Note that the effect of the backdrop-filter will not be visible unless some
```

Comment:
Make this paragraph into a non-normative note (there is a way to do that in bikeshed)..

Reply:
Done.